### DISCUSSION TITLE
Installation wiki.js

### DISCUSSION THREAD

**Discussion by VerenaZernpfennig** [requarks/wiki#3428] [Help / Questions]
**Question**
Hi, maybe I just don't get it. I downloaded the tar.gz for Windows 10, extract all the files and setup the properties for the connection in config.yml. My port of my localhost is 8080 and the database connection should also work. I didn't change any other properties in this file. But when I start localhost:8080/docwiki/server (docwiki is the name of the folder) I only see the file structure but no further possibilities. I tried to do this "node server" installation step but unfortunately "node" is not recognized by my powershell as a command. Do you have an idea what I am doing wrong?

Thanks in advance.
Kind regards
Verena

**Host Info (please complete the following information):**
OS: [Windows 10, XAMPP]
Wiki.js version: [2.4.107]
Database engine: [mysql 5.7]

**Comment by coatmaker618** [comment]
I'm concerned that you see the file structure on port 8080! What is serving those files?

However, `node server` is the correct way to run this.  That said, that exact command is making assumptions and can be done more explicitly.

1. `node` is telling nodejs to run, and it's assuming the command node (pointing to node js) is in your PATH (as an Environment Variable).  If nodejs is installed, you can either add it to your path or find the full path (eg: "C:\Program Files\nodejs\node") as use that as the first part of your command, replacing `node` but still followed by the 2nd part, `server`.

If you have not installed it, installation information for nodejs can be found here:
https://docs.requarks.io/install/requirements

2. `server` is a folder in wikijs that contains code that you want nodejs to run.  The easy way to do this is to make sure you run the `node server` command in the location of the server folder (eg: the docwiki installation folder).

**Comment by Smankusors** [comment]
I see you used XAMPP, but that doesn't mean you copy pasted the Wiki.js to htdocs. Wiki.js is not written in PHP, but Node.js. And Wiki.js runs it's own web server. You don't need XAMPP for that.

If you still want to use XAMPP, you can setup reverse proxy on your Apache2.
